Makefile

```diff
@@ -0,0 +1,29 @@
+DESTDIR = /usr/local
```

Comment:
This should be PREFIX, not DESTDIR, BINDIR and MANDIR should use PREFIX instead of DESTDIR, and then $(DESTDIR) should be prepended to $(BINDIR) and $(MANDIR) when they're used in targets, so DESTDIR is still used even if the user manually sets BINDIR or MANDIR to something else